recipes/geoschem-gcpy/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "geoschem-gcpy" %}
+{% set version = "1.0.0b0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geoschem-gcpy-{{ version }}.tar.gz
+  sha256: c73e06ef73f0aeacc8cc092546c3826096fb5a428648db9ff33cc838f603ded4
+
+build:
+  number: 0
+  skip: true  # [win or py<36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - numpy 1.19.1
+    - cartopy >=0.17.0
+  run:
+    - cartopy >=0.17.0
+    - dask >=2.3.0
+    - esmpy >=7.1.0
+    - h5py >=2.9.0
+    - h5netcdf
+    - joblib >=0.17.0
+    - matplotlib-base >=3.1.1
+    - netcdf4 >=1.4.2
+    - {{ pin_compatible('numpy') }}
+    - pandas >=0.25.1
+    - pypdf2 >=1.26.0
+    - python
+    - scipy >=1.3.1
+    - sphinx
+    - sphinx_rtd_theme
+    - sphinx-autoapi
+    - tabulate >=0.8.3
+    - xarray >=0.15.0
+    - xbpch >=0.3.5
+    - xesmf >=0.2.1
+
+test:
+  imports:
+    - gcpy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/geoschem/gcpy
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Python toolkit for working with GEOS-Chem output and input'
+  doc_url: https://gcpy.readthedocs.io/en/latest/
+  dev_url: https://github.com/geoschem/gcpy
+
+extra:
+  recipe-maintainers:
+    - WilliamDowns
+    - yantosca
+    - lizziel
+    - msulprizio
+    - LiamBindle
```

Comment:
@WilliamDowns we need all these folks to confirm that they are OK being maintainers of this recipe.

Reply:
@ocefpaf All maintainers have now confirmed below.